spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsAutoConfiguration.java

```diff
@@ -76,12 +77,13 @@
 @EnableConfigurationProperties(MetricsProperties.class)
 @Import({ MeterBindersConfiguration.class, WebMvcMetricsConfiguration.class,
 		WebFluxMetricsConfiguration.class, RestTemplateMetricsConfiguration.class,
-		DataSourcePoolMetricsConfiguration.class, AtlasExportConfiguration.class,
-		DatadogExportConfiguration.class, GangliaExportConfiguration.class,
-		GraphiteExportConfiguration.class, InfluxExportConfiguration.class,
-		JmxExportConfiguration.class, PrometheusExportConfiguration.class,
-		SimpleExportConfiguration.class, StatsdExportConfiguration.class })
-@AutoConfigureAfter({DataSourceAutoConfiguration.class, RabbitAutoConfiguration.class})
+		CacheMetricsConfiguration.class, DataSourcePoolMetricsConfiguration.class,
```

Comment:
This adds `CacheMetricsConfiguration`, which probably should have been present already independently of RabbitMQ metrics?

Reply:
I may have made something wrong when merging with master, but it seems `CacheMetricsConfiguration` was already there: https://github.com/spring-projects/spring-boot/blob/027e6baba53a83e385bc307925c654c55da346a8/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsAutoConfiguration.java#L73. Am I missing something?

Reply:
Ah, ok. Great.